mycore-orcid2/src/main/java/org/mycore/orcid2/user/MCRORCIDUser.java

```diff
@@ -72,13 +73,28 @@ public class MCRORCIDUser {
 
     private final MCRUser user;
 
+    private final MCRLegalEntityService legalEntityService;
+
     /**
      * Wraps MCRUser to MCRORCIDUser.
      *
      * @param user the MCRUser
      */
     public MCRORCIDUser(MCRUser user) {
         this.user = user;
+        legalEntityService = MCRConfiguration2.getInstanceOfOrThrow(
+            MCRLegalEntityService.class, MCRORCIDConstants.CONFIG_PREFIX + "LegalEntityService.Class");
```

Comment:
Would it perhaps make more sense to have a centralized service? Something like a `DefaultLegalEntityService` where you can configure the default service? There are certainly other places where this service is needed.